seacrowd/utils/common_parser.py

```diff
@@ -6,7 +6,7 @@
 
 def load_conll_data(file_path):
     # Read file
-    data = open(file_path, "r").readlines()
+    data = open(file_path, "r", encoding="utf8").readlines()
```

Comment:
can you make a separate PR for this? since this is a shared utility.

Reply:
Sure

Reply:
created https://github.com/SEACrowd/seacrowd-datahub/pull/328

Reply:
in that case, mind removing changes on `common_parser.py` in this PR? we'll address that on your newly PR instead